Sources/Document/Document.swift

```diff
@@ -50,10 +54,48 @@ class Document {
             self.localChanges.append(change)
             self.changeID = change.getID()
 
+            let changeInfo = ChangeInfo(change: change, paths: self.createPaths(change: change))
+            let changeEvent = LocalChangeEvent(value: [changeInfo])
+            self.eventStream.send(changeEvent)
+
             Logger.trivial("after update a local change: \(self.toJSON())")
         }
     }
 
+    /**
+     * `applyChangePack` applies the given change pack into this document.
+     * 1. Remove local changes applied to server.
+     * 2. Update the checkpoint.
+     * 3. Do Garbage collection.
+     *
+     * - Parameter pack: change pack
+     */
+    func applyChangePack(pack: ChangePack) throws {
```

Comment:
@hackerwins comments above function ```applyChangePack``` and comments in function ```applyChangePack``` do not same in JS-SDK. Please check it.

### The comments above function ```applyChangePack```
```
   * 1. Remove local changes applied to server.
   * 2. Update the checkpoint.
   * 3. Do Garbage collection.
```
### the comments in function ```applyChangePack```
```
// 02. Remove local changes applied to server.
    while (this.localChanges.length) {
      const change = this.localChanges[0];
      if (change.getID().getClientSeq() > pack.getCheckpoint().getClientSeq()) {
        break;
      }
      this.localChanges.shift();
    }

    // 03. Update the checkpoint.
    this.checkpoint = this.checkpoint.forward(pack.getCheckpoint());

    // 04. Do Garbage collection.
```